controller/rpc/scanner.go

```diff
@@ -332,13 +404,13 @@ func ScanAwsLambdaFunc(ctx context.Context, funcInput *share.CLUSAwsFuncScanInpu
 		ScanSecrets: true, // default: always
 	}
 
-	client, err := getScannerServiceClient(scanner, true)
+	client, scanner, err := ScannerMgr.getScannerServiceClient(scanner, true)
 	if err != nil {
 		err := fmt.Errorf("No scan client available")
 		log.WithFields(log.Fields{"error": err}).Error()
 		return nil, err
 	}
-	defer decScanningCount(scanner)
+	defer ScannerMgr.releaseScannerAvailability(scanner)
```

Comment:
This line should be moved to after `scanner := ScannerMgr.acquireScannerAvailability()`, otherwise the scanner availability would not be increased again if it returns between two calls.